motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -105,6 +108,9 @@ JointTrajectoryAction::JointTrajectoryAction() :
   pub_trajectory_command_ = node_.advertise<motoman_msgs::DynamicJointTrajectory>(
                               "joint_path_command", 1);
 
+  sub_trajectory_state_all_  = this->node_.subscribe<control_msgs::FollowJointTrajectoryFeedback>("/feedback_states", 1,
```

Comment:
subscriber **_sub_trajectory_state__** listen to single group feedback, for example **_*/arm_left/feedback_states_** and **_*/arm_right/feedback_states_**;
but when we execute a multi-groups motion, such as group "arms" ( = "arm_left" + "arm_right"), we should listen to  **_/feedback_states_** for all joints.
The origin code ignores feedback of multi-groups motion, and the robot state "LOST".
In this PR, I create a new subscriber **_sub_trajectory_state_all__** to fix this problem.